web/src/components/AttributeList/AttributeList.tsx

```diff
@@ -1,17 +1,30 @@
-import {TSpanFlatAttribute} from '../../types/Span.types';
-import AttributeRow from '../AttributeRow';
+import AttributeRow from 'components/AttributeRow';
+import {IResultAssertions} from 'components/SpanDetail/SpanDetail';
+import {TSpanFlatAttribute} from 'types/Span.types';
 import * as S from './AttributeList.styled';
 
-interface IAttributeListProps {
+interface IProps {
+  assertions?: IResultAssertions;
   attributeList: TSpanFlatAttribute[];
   onCreateAssertion(attribute: TSpanFlatAttribute): void;
 }
 
-const AttributeList: React.FC<IAttributeListProps> = ({attributeList, onCreateAssertion}) => {
+const AttributeList: React.FC<IProps> = ({assertions, attributeList, onCreateAssertion}) => {
+  const onCopy = (value: string) => {
+    navigator.clipboard.writeText(value);
+  };
```

Comment:
Moving the `onCopy` function here to avoid creating a function for each attribute row.